packages/components/src/internal/components/search/utils.ts

```diff
@@ -938,6 +994,11 @@ export function getDataTypeFiltersWithNotInQueryUpdate(
     return dataTypeFiltersUpdated;
 }
 
+function getNotInLabKeySql(schemaQuery: SchemaQuery, targetQueryFilterKey: string, cf?: Query.ContainerFilter): string {
```

Comment:
The name here doesn't match the query. I was expecting a `NOT IN (...)` query, but it's a `NOT NULL` query.